src/main/java/com/example/graduatework/service/impl/ImageServiceImpl.java

```diff
@@ -0,0 +1,81 @@
+package com.example.graduatework.service.impl;
+
+import com.example.graduatework.service.ImageService;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.lang.String;
+import java.util.Objects;
+
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class ImageServiceImpl implements ImageService {
+    @Value("${image.dir.path}")
+    private String imageDir;
+
+
+    /**
+     * Загрузка изображения
+     */
+
+    @Override
+    public String uploadImage(String imageId, MultipartFile imageFile) throws IOException {
+        String imageName = imageId + "." + getExtensions(Objects.requireNonNull(imageFile.getOriginalFilename()));
+
+        Path filePath = Path.of(String.format("%s/%s", imageDir, imageName));
+        Files.createDirectories(filePath.getParent());
+        Files.deleteIfExists(filePath);
+
+        Files.write(filePath, imageFile.getInputStream().readAllBytes());
+        return imageName.replace(".", "");
```

Comment:
```suggestion
        return imageId;
```
надо возвращать, а не imageName